libyul/optimiser/StructuralSimplifier.cpp

```diff
@@ -45,6 +45,27 @@ void StructuralSimplifier::operator()(Block& _block)
 	popScope();
 }
 
+bool StructuralSimplifier::isConstExpression(Expression const& _expression, u256& _out_value)
```

Comment:
```suggestion
boost::optional<u256> StructuralSimplifier::isConstExpression(Expression const& _expression)
```

I'm also not too sure about the name - it can be constant even though we don't know the numeric value...